cosmos/carbon.json

```diff
@@ -251,9 +251,9 @@
       "coinDecimals": 18,
       "coinGeckoId": "canto",
       "gasPriceStep": {
-        "low": 153846153846.15384,
-        "average": 192307692307.6923,
-        "high": 230769230769.23077
+        "low": 66666666666.666666666667,
+        "average": 66666666666.666666666667,
+        "high": 66666666666.666666666667
```

Comment:
Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas. Also, follow the original number, round to 5 d.p.